sbi/utils/conditional_density.py

```diff
@@ -107,11 +120,21 @@ def conditional_corrcoeff(
         resolution: Number of grid points on which the conditional distribution is
             evaluated. A higher value increases the accuracy of the estimated
             correlation but also increases the computational cost.
+        warn_about_deprecation: With sbi v0.15.0, we depracated the import of this
+            function from `sbi.utils`. Instead, it should be imported from
+            `sbi.analysis`.
 
     Returns: Average conditional correlation matrix of shape either `(num_dim, num_dim)`
     or `(len(subset), len(subset))` if `subset` was specified.
     """
 
+    if warn_about_deprecation:
+        warn(
+            "Importing `conditional_corrcoeff` from `sbi.utils` is deprecated since "
+            "sbi v0.15.0. Instead, use "
+            "`from sbi.analysis import conditional_corrcoeff`."
+        )
+
     condition = ensure_theta_batched(condition)
```

Comment:
This comment applies to all `sbi/utils/` functions that got moved:
Couldn't we just delete the code from the function and use all args and kwargs and dispatch to the function in the new location? That way, we avoid needing to keep anything in sync as long as the pointer is still in place

Reply:
I tried that, but it does not work. When doing that and calling
```python
from sbi.analysis import pairplot
```
will trigger a circular import because `sbi.analysis` has to import helpers from `sbi.utils`. Maybe it's possible to figure out some magic order of imports to make it work, but I do not think that it's worth the (rather small) overhead of keeping things in sync.

Reply:
Alright, makes sense to keep it as it is then